process/accountProcessor.go

```diff
@@ -229,7 +229,7 @@ func (ap *AccountProcessor) GetNFTTokenIDsRegisteredByAddress(address string, op
 		log.Error("account get owned NFTs", "observer", observer.Address, "address", address, "error", err.Error())
 	}
 
-	return nil, ErrSendingRequest
+	return nil, fmt.Errorf("%w, %s", ErrSendingRequest, apiResponse.Error)
```

Comment:
if each observer is offline, all `apiResponse` instances will be empty, so the returned error will be: `error sending request, `
I would suggest creating a new function `wrapObserverError` and only append the observer's error if not empty. 

^
valid for the entire pr

Reply:
pushed